docs/building-chain/app-go.md

```diff
@@ -0,0 +1,20 @@
+<!--
+order: 0
+-->
+
+# `app.go` Overview
+
+This section is intended to provide an overview of the `app.go` file and is still a work in progress.
+For now we invite you to read the [tutorials](https://tutorials.cosmos.network) for a deep dive on how to build a chain.
+
+## `app.go`
+
+Since `v0.47.0` the Cosmos SDK have made easier wiring an `app.go` thanks to dependency injection:
+
++++ https://github.com/cosmos/cosmos-sdk/blob/main/simapp/app_config.go
+
++++ https://github.com/cosmos/cosmos-sdk/blob/main/simapp/app.go
```

Comment:
We should open an issue to fill this out. Generally not a fan of just linking code as it doesn't explain why this and why that. This wouldn't provide much info to a new user.

Reply:
As docs.cosmos.network default on v0.46, a new user won't see it until he digs.
I can remove the code, but I am really for to keep these files, as they allow me to already link to them.

Reply:
Totally fine it keep them, mainly thinking it would be useful to also break it down